can/util.py

```diff
@@ -4,8 +4,11 @@
 """
 from __future__ import absolute_import
 
+from pluggy import PluginManager
+
 import can
-from can.interfaces import VALID_INTERFACES
+from can.interfaces import VALID_INTERFACES, hookspecs
```

Comment:
Missing import (`hookspecs`) breaks the build - https://travis-ci.org/hardbyte/python-can/jobs/328610852